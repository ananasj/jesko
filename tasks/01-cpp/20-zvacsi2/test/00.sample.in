5
2 6 4 1 1
10
