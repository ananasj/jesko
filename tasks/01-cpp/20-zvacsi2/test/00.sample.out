12 16 14 11 11 
